model/minio/minio_cluster.rb

```diff
@@ -60,4 +67,12 @@ def single_instance_multi_drive?
   def hostname
     "#{name}.#{Config.minio_host_name}"
   end
+
+  def root_certs
+    root_cert_1.to_s + root_cert_2.to_s
+  end
+
+  def protocol
+    root_certs.empty? ? "http" : "https"
```

Comment:
do we ever expect the "http" case to happen in prod really? I know we have some path dependency, but it would be a good idea to make sure it's always https when in production mode unless we really think people should degrade to http in this way.

Reply:
No, it wouldn't happen but the main reason I wanted to keep it is to not break any operation that's going on with the existing minio cluster. We will remove this all together after the new minio cluster is operational.